docs/building-apps.md

```diff
@@ -102,6 +102,13 @@ It's easiest to begin from the start with TypeScript by using the `typescript` t
 npm install --save typescript @types/react @types/react-dom @types/jest
 ```
 
+Note: If you receive an error when building the app that says `Cannot find module 'typescript'`, try to set `NODE_PATH` to point global
```

Comment:
This note is a common thing for opt-in modules but it appears 3 times. Is it possible to note once for all?

Reply:
OK, I'm on checking.

Reply:
@MikyungKim 
Please check the fixed comment.